byterun/pyvm2.py

```diff
@@ -566,6 +594,20 @@ def byte_DELETE_SUBSCR(self):
 
     ## Building
 
+    def byte_BUILD_TUPLE_UNPACK_WITH_CALL(self, count):
+        # This is similar to BUILD_TUPLE_UNPACK, but is used for f(*x, *y, *z)
+        # call syntax. The stack item at position count + 1 should be the
+        # corresponding callable f.
+        elts = self.popn(count)
+        self.push(tuple(e for l in elts for e in l))
+
+    def byte_BUILD_TUPLE_UNPACK(self, count):
```

Comment:
in the C source, BUILD_TUPLE_UNPACK, BUILD_TUPLE_UNPACK_WITH_CALL, and BUILD_LIST_UNPACK have the same target, should we reflect that by having them share the same target here?

Reply:
I will update to reflect that that.